about/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Об авторе</title>
+</head>
+<body>
+    <h2>Немного обо мне</h2>
+    <p>Добрый день. <strong>Меня зовут Алексей,</strong><em>мне 31 год.</em> <mark>Экономическое образование.</mark><br>
+        Долгое время работаю в тренажерном зале тренером и являюсь выступающим <br>
+        атлетом по силовым видам спорта &#128170;. <del>В связи с последними событиями, <br>
+        происходящими в моей стране,</del> &#128532; <ins>решил сменить сферу деятельности,</ins><br>
+        на ту в в которой <sub>была бы возможность</sub><sup>работать дистанционно.</sup><br>
+        <q>Одна цитата</q></p>
+    <h3>Почему Front-end</h3>
+    <p>Я выбрал направление Front-end разработки т.к. нравится<br> 
+        видить результат своей работы сразу и эксперементировать.</p>
+    <h4>Cильные качества</h4>
+    <li>Могу есть одной рукой</li>
+    <li>Могу слышать в темноте</li>
+    <li>ICQ: выше среднего</li>
+    <p>ссылка на мой профиль в <a href="https://github.com/Tandaral" target="_blanck">GitHub</a></p>
```

Comment:
I would recommend using _blank instead of _blanck. it is advisable to fix this in a future pull request. The rest works fine. 👍